docs/examples/index.rst

```diff
@@ -0,0 +1,118 @@
+directly in Colab.
+
+Image classification
+********************
+
+- `MNIST <https://github.com/google/flax/tree/main/examples/mnist/>`__ -
+  `🕹Interactive🕹 <https://colab.research.google.com/github/google/flax/blob/main/examples/mnist/mnist.ipynb>`__:
+  Convolutional neural network for MNIST classification (featuring simple
+  code).
+
+- `ImageNet <https://github.com/google/flax/tree/main/examples/imagenet/>`__ -
+  `🕹Interactive🕹 <https://colab.research.google.com/github/google/flax/blob/main/examples/imagenet/imagenet.ipynb>`__:
+  Resnet-50 on ImageNet with weight decay (featuring multi host SPMD, custom
+  preprocessing, checkpointing, dynamic scaling, mixed precision).
+
+Reinforcement learning
+**********************
+
+- `Proximal Policy Optimization <https://github.com/google/flax/tree/main/examples/ppo/>`__:
+  Learning to play Atari games (featuring single host SPMD, RL setup).
+
+Natural language processing
+***************************
+
+-  `Sequence to sequence for number
+   addition <https://github.com/google/flax/tree/main/examples/seq2seq/>`__:
+   (featuring simple code, LSTM state handling, on the fly data generation).
+-  `Parts-of-speech
+   tagging <https://github.com/google/flax/tree/main/examples/nlp_seq/>`__: Simple
+   transformer encoder model using the universal dependency dataset.
+-  `Sentiment
+   classification <https://github.com/google/flax/tree/main/examples/sst2/>`__:
+   with a LSTM model.
+-  `Transformer encoder/decoder model trained on
+   WMT <https://github.com/google/flax/tree/main/examples/wmt/>`__:
+   Translating English/German (featuring multihost SPMD, dynamic bucketing,
+   attention cache, packed sequences, recipe for TPU training on GCP).
+-  `Transformer encoder trained on one billion word
+   benchmark <https://github.com/google/flax/tree/main/examples/lm1b/>`__:
+   for autoregressive language modeling, based on the WMT example above.
+
+Generative models
+*****************
+
+-  `Variational
+   auto-encoder <https://github.com/google/flax/tree/main/examples/vae/>`__:
+   Trained on binarized MNIST (featuring simple code, vmap).
+
+Graph modeling
+**************
+
+- `Graph Neural Networks <https://github.com/google/flax/tree/main/examples/ogbg_molpcba/>`__:
+  Molecular predictions on ogbg-molpcba from the Open Graph Benchmark.
+
+
+Contributing Examples
+**********************
+
+Most of our examples in this directory follow a structure that we found to work
+well with Flax projects, and we strive to make the examples easy to explore and
+easy to fork. In particular (taken from `#231 <https://github.com/google/flax/issues/231>`__)
+
+- README: contains links to paper, command line, `TensorBoard <https://tensorboard.dev/>`__ metrics
+- Focus: an example is about a single model/dataset
+- Configs: we use ``ml_collections.ConfigDict`` stored under ``configs/``
+- Tests: executable ``main.py`` loads ``train.py`` which has ``train_test.py``
+- Data: is read from [TensorFlow Datasets]
```

Comment:
dead link

Reply:
Curiously this in currently broken in `examples/README.md`, will add a link.